pkg/config/generate.go

```diff
@@ -11,6 +11,7 @@ import (
 	"os"
```

Comment:
This file getting enormous. Maybe worth splitting it to separate files like `generate.go` - only generate function, `images.go` - image resolved `resolvers.go/hellpers.go` for all stuff like `selectNodeImage` `selectContainerImages` ? 

Not in this PR but I would like to see it split :/

Reply:
agree

Reply:
https://github.com/openshift/openshift-azure/issues/248